2-mandatory/3-stocks.js

```diff
@@ -35,8 +35,25 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
 */
 function getAveragePrices(closingPricesForAllStocks) {
     // TODO
+
+   let output=[]
+
+    for (let i=0 ; i< closingPricesForAllStocks.lenght; i++){
+        const price = closingPricesForAllStocks[i];
```

Comment:
It is worth thinking about the parameter `closingPricesForAllStocks`. What it is the type of this variable?
From the question above we can see: "it's an array of arrays". So `closingPricesForAllStocks[i]` is not just one price, it's an array of all the prices for one stock.